bpo-33043: Add a Contributing to Docs link and Update the Found a Bug Page

suhearsawho opened the pull request:
A 'Contributing to Docs' link was added to the bottom of the docs.python.org page. Along with this, the 'Found a Bug' (https://docs.python.org/3/bugs.html) page was edited as the table in the Documentation Bugs section was changed (2 extra links were added and summaries were revised/added). 


<!-- issue-number: [bpo-33043](https://bugs.python.org/issue33043) -->
https://bugs.python.org/issue33043
<!-- /issue-number -->

suhearsawho committed:
changes to html file -> added contributing to docs link at the end of the page

suhearsawho committed:
revisions to the dealing with bugs page. added more links in the documentation bugs section

blurb-it[bot] committed:
📜🤖 Added by blurb_it.

willingc commented on Doc/bugs.rst:
```diff
@@ -25,7 +25,15 @@ docs@python.org (behavioral bugs can be sent to python-list@python.org).
-   `Documentation bugs`_ on the Python issue tracker
+
+   `Documentation bugs`_
+      Link to the Python issue tracker. Provides a list of documentation bugs that have been submitted.
```
```suggestion
      A list of documentation bugs that have been submitted to the Python issue tracker.
```

suhearsawho replied:
Thank you for the recommendation! I have added the changes.

willingc reviewed (commented):
Thanks for the PR @suhearsawho. I have left a suggestion as an alterative to "Link to"

willingc committed:
Update Doc/bugs.rst

Updated Doc/bugs.rst in accordance with willingc and JulienPalard suggestions.

Co-Authored-By: suhearsawho <susansu.software@gmail.com>

willingc reviewed (approved):
Thanks @suhearsawho for the contribution.
